editor/src/clj/editor/debug_view.clj

```diff
@@ -414,8 +414,8 @@
      (when (or (seq added) (seq removed))
        (mobdebug/with-suspended-session debug-session
          (fn [debug-session]
-           (run! #(set-breakpoint! debug-session %) added)
-           (run! #(remove-breakpoint! debug-session %) removed)))))))
+           (run! #(remove-breakpoint! debug-session %) removed)
+           (run! #(set-breakpoint! debug-session %) added)))))))
```

Comment:
Need to remove before adding e.g. when we modify a breakpoint condition: we want to remove-then-set updates (as expected), but set-then-remove removes (unexpected)